version.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "v0.2.19"
+  "version": "v0.2.20"
```

Comment:
should this be v0.3.0?

Reply:
Since one of the core's valuestore wrappers has a slight API change in that the options now come from the underlying implementation, that would be a breaking change, so yes.